wp-content/themes/csisnuclearnetwork/assets/_scss/components/_facet.scss

```diff
@@ -0,0 +1,306 @@
+    position: relative;
+    background-color: var(--input-background);
+    border: 2px solid var(--input-border);
+    border-radius: 3px;
+    cursor: default;
+    transition: background 0.3s ease-in-out, border 0.3s ease-in-out;
+
+    &:hover {
+      filter: $shadow--2;
+    }
+  }
+
+  &-label-wrap,
+  &-dropdown {
+    user-select: none;
+  }
+
+  &-label {
+    position: relative;
+    height: 100%;
+    padding: rem(16);
+    overflow: hidden;
+    color: $color-white-100;
+    @extend %text-style-label-large;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    word-break: break-word;
+
+    .fs-default & {
+      color: $color-black-100;
+    }
+  }
+
+  .fs-label {
+    @extend %text-style-label-large;
+    color: $color-black-190;
+  }
+
+  &-arrow {
+    position: absolute;
+    top: 0;
+    right: rem(16);
+    bottom: 0;
+    width: 0;
+    height: 0;
+    margin: auto;
+    border-top: 5px solid var(--arrow-color);
+    border-right: 5px solid transparent;
+    border-left: 5px solid transparent;
+    transition: ease-in 0.15s;
+
+    .fs-open & {
+      transform: rotate(-180deg);
+    }
+  }
+
+  &-dropdown {
+    position: absolute;
+    z-index: 1000;
+    width: 100%;
+    margin: 0;
+    background-color: #fff;
+    border: 1px solid #ddd;
+
+    .fs-options {
+      max-height: 260px;
+      overflow: auto;
+    }
+  }
+
+  &-search {
+    padding: 0 8px;
+    border-bottom: 1px solid #eee;
+
+    input {
+      width: 100%;
+      padding: 6px 0;
+      border: 0;
+      outline: none;
+      box-shadow: none !important;
+    }
+  }
+
+  &-option,
+  &-search,
+  &-optgroup-label,
+  &-no-results {
+    padding: rem(12);
+    cursor: default;
+  }
+
+  &-option {
+    --checkbox-bg: #{$color-postbg-200};
+    --checkbox-border: #{$color-border-dark-115};
+    position: relative;
+    display: flex;
+    gap: rem(12);
+    line-height: 1.15;
+    word-break: break-all;
+    cursor: pointer;
+
+    &:hover {
+      background: $color-bg-light-200;
+
+      i {
+        border-width: 2px;
+      }
+    }
+
+    &.selected {
+      --checkbox-bg: #{$color-accent-primary-blue-200};
+      --checkbox-border: #{$color-border-light-115};
+
+      i::before {
+        --borderWidth: 2px;
+        --height: 10px;
+        --width: 5px;
+        --borderColor: #{$color-white-190};
+        content: '';
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        display: block;
+        width: var(--width);
+        height: var(--height);
+        border-right: var(--borderWidth) solid var(--borderColor);
+        border-bottom: var(--borderWidth) solid var(--borderColor);
+        transform: translate(-50%, -60%) rotate(45deg);
+      }
+    }
+
+    &:last-child {
+      border-bottom: 0;
+    }
+
+    &.disabled {
+      cursor: default;
+      opacity: 0.4;
+    }
+
+    &.d1 .fs-option-label {
+      padding-left: 0 !important;
+    }
+
+    &.d1 .fs-checkbox {
+      padding-left: rem(20px);
+    }
```

Comment:
I wasn't able to find anything with a class of `d1`, but please let me know if I'm missing something.